docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+
+## How to Use?
+The tuple type in the abstract class is a generic parameter. Concrete subclasses need to choose an appropriate class (such as String or java class for [POJO](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) etc for the generic parameter) and also implement a couple of abstract methods: getTuple(ResultSet) to parse database row to chosen class and emitTuple(T) to emit the tuple.
+
+In principle, no ports need be defined in the rare case that the operator simply writes tuples to some external sink or merely maintains aggregated statistics. But in most common scenarios, the tuples need to be sent to one or more downstream operators for additional processing such as parsing, enrichment or aggregation; in such cases, appropriate output ports are defined and the emitTuple(T) implementation dispatches tuples to the desired output ports.
+
+Couple of concrete implementations are provided in Malhar:
+
+1. [JdbcPOJOPollInputOperator](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/com/datatorrent/lib/db/jdbc/JdbcPOJOPollInputOperator.java): It uses java Object for the generic parameter. You need to set output port attribute TUPLE_CLASS to define your POJO class name to define Object type. The record fetched from database table will be parsed as object of configured POJO class. This operator defines a single output port and processes each database table record one by one as a tuple object. Details are discussed below.
+
+2. [JdbcPollInputOperator](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/com/datatorrent/lib/db/jdbc/JdbcPollInputOperator.java): It uses String for the generic parameter. This operator defines a single port and processes each database table record one by one as String tuple. Details are discussed below.
+
+## Partitioning of JDBC Poller
+#### Static Partitioning
+Only static partitioning is supported for JDBC Poller Input Operator. Configure parameter `partitionCount` to define the desired number of initial partitions (4 in this example).
+**Note**: The additional partition will be created to poll newly added records.
+
+```xml
+  <property>
+    <name>dt.operator.{OperatorName}.prop.partitionCount</name>
+    <value>4</value>
+  </property>
+```
+
+where {OperatorName} is the name of the JDBC Poller operator.
+
+This will create 5 operator instances in all. Four of these will fetch the data which is currently in the table. We call these static non-polling partitions. An additional partition will be created which will read any newly added data. We call such a partition as a polling partition, as it "polls" for newly added data. Changing the value of the property will create corresponding number of static partitions.
```

Comment:
Omit sentence: "Changing the value ..." since it seems redundant. Some questions arise naturally here:
1. Will there always be one and only one polling partition ?
2. Once the 4 partitions finish reading current table contents, are they idle (since all newly added data is
    read only by the polling partition) ?